### PR TITLE
👷 One CI file to rule them all

### PR DESCRIPTION
**Pull request by coyotte508**

### CODE REVIEW

File: packages/inference/test/global.d.ts [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-		interface Assertion extends CustomMatchers {}
-		interface AsymmetricMatchersContaining extends CustomMatchers {}
+		interface Assertion {
+			// todo: fix the any
+			closeTo(expected: number, precision: number): any;
+		}
+		interface AsymmetricMatchersContaining {
+			// todo: fix the any
+			closeTo(expected: number, precision: number): any;
+		}
```
Comment (coyotte508) [inline packages/inference/test/global.d.ts:10]: to pass linting cc @pngwn

File: package.json [hunk 2]
```diff
@@ -7,11 +7,12 @@
...
 		"eslint-plugin-prettier": "^4.2.1",
 		"prettier": "^2.8.4",
-		"typescript": "^4.9.5"
+		"typescript": "^4.9.5",
+		"vite": "^4.1.4",
+		"vitest": "^0.29.2"
 	},
 	"scripts": {
 		"format": "prettier --write .eslintrc.cjs && eslint --quiet --fix --ext .cjs,.ts .eslintrc.cjs",
-		"test:run": "vitest run",
```
Comment (coyotte508) [inline package.json:14]: This was giving this error message: 
image

File: packages/inference/test/global.d.ts (hunk 1 above)
Comment (pngwn) [reply packages/inference/test/global.d.ts:10]: exhibit a of why i hate linting :P

Comment (pngwn) [reply packages/inference/test/global.d.ts:10]: This isn't the same btw but works well enough

**Pull request merged**